src/AllLayers.js

```diff
@@ -133,6 +137,14 @@ L.LayerGroup.environmentalLayers = L.LayerGroup.extend(
         // Update map state from hash
         hash.update(this.options.currentHash);
       }
+        
+      for (let layer of this.options.layers.include) {
+        if (!this.options.addLayersToMap) {
```

Comment:
How did this logic got removed?

Reply:
These lines were added. The ones that were removed are here: https://github.com/publiclab/leaflet-environmental-layers/blame/00c10f2c157a90a1d98a0254a061f9d6dd8f4c57/src/AllLayers.js#L123-L125